clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -1308,6 +1309,57 @@ export const CustomDatePicker = ({
           size="sm"
           focusBorderColor="primary.600"
           data-testid={`input-${name}`}
+          disabled={isDisabled}
+        />
+        <ErrorMessage
+          isInvalid={isInvalid}
+          message={meta.error}
+          fieldName={field.name}
+        />
+      </VStack>
+    </FormControl>
+  );
+};
+
+export const CustomDateTimeInput = ({
+  label,
+  name,
+  tooltip,
+  disabled,
+  isRequired,
+  ...props
+}: CustomInputProps & FieldHookConfig<string>) => {
+  const [field, meta, { setValue, setTouched }] = useField(name);
+  const isInvalid = !!(meta.touched && meta.error);
+
+  const { validateField } = useFormikContext();
+
+  return (
+    <FormControl isRequired={isRequired} isInvalid={isInvalid}>
+      <VStack align="start">
+        {!!label && (
+          <Flex align="center">
+            <Label htmlFor={props.id || name} fontSize="xs" my={0} mr={1}>
```

Comment:
This `htmlFor` isn't actually doing anything, as far as I can tell. To be valid HTML, this `for` attribute needs to be the same as the `id` attribute value on the form field. The `<Input>` below isn't defining an `id` at all. The `for` attribute does not work with the `name` attribute.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for

Unfortunately, it looks like this bad pattern exists in a lot of places in our app so we'll need to clean those up eventually.